src/admin-display.cls.php

```diff
@@ -50,6 +50,14 @@ public function __construct() {
 		// Main js
 		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
 
+		//sidebar meta box
+		/* Add meta boxes on the 'add_meta_boxes' hook. */
+		add_action( 'add_meta_boxes', array( Sidebar::cls(), 'add_meta_boxes' ) );
```

Comment:
if you are reusing same instance, set it to a variable before using to avoid multi function call for efficiency concern. e.g. `$__sidebar = $this->cls('Sidebar');`

Reply:
add_action( 'add_meta_boxes', $__sidebar = $this->cls('Sidebar')->add_meta_boxes() ) );
like this ???

Reply:
add_action( 'add_meta_boxes', $__sidebar = $this->cls('Sidebar')->add_meta_boxes() ) );
like this ???

Reply:
```
$__sidebar = $this->cls('Sidebar');
add_action( 'add_meta_boxes', array($__sidebar, 'add_meta_boxes') ) );
```